examples/angular/src/app/components/content/content.component.ts

```diff
@@ -48,6 +53,8 @@ export class ContentComponent implements OnInit, OnDestroy {
       this.getAccount(),
     ]);
 
+    this.verifyMessageBrowserWallet();
```

Comment:
After we make `verifyMessageBrowserWallet` async we need to add the `await` keyword and move this call down below the `this.subscribeToEvents();` to avoid blocking the page-load.